src/index.html

```diff
@@ -11,6 +11,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input data-qa="keypress"
+         class="search-form__input search-form__input_big" type="text"
+         placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
        <input 
          data-qa="keypress"
          class="search-form__input search-form__input_big" 
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
       >
```
Consider fixing indents according to the code style guide